src/main.cpp

```diff
@@ -199,6 +199,9 @@ void vcMain_LoadSettings(vcState *pProgramState, bool forceDefaults)
     case 1: ImGui::StyleColorsDark(); break;
     case 2: ImGui::StyleColorsLight(); break;
     }
+
+    SDL_SetWindowSize(pProgramState->pWindow, pProgramState->settings.window.width, pProgramState->settings.window.height);
```

Comment:
I can't remember the exact rules for this- there was problems doing it this way originally. From memory- you want it to happen at start-up but not on force reset?

Reply:
This resolves AB#97